api/authorization.yml

```diff
@@ -156,6 +156,8 @@ components:
           format: byte
         external_id:
           type: string
+        invite:
```

Comment:
Please document this field. It is not clear to me whether this is imperative or descriptive.

Reply:
Done